pkg/platform/model/inventory.go

```diff
@@ -271,7 +282,21 @@ func filterPlatformIDs(hostPlatform, hostArch string, platformIDs []strfmt.UUID)
 		return fallback, nil
 	}
 
-	return nil, &ErrNoMatchingPlatform{hostPlatform, hostArch}
+	return nil, &ErrNoMatchingPlatform{hostPlatform, hostArch, libcVersion}
+}
+
+func fetchLibcVersion() string {
+	var libcVersion string
```

Comment:
What do you think about simplifying this to:

```
if runtime.GOOS != "linux" {
    return ""
}

libcInfo, err := sysinfo.Libc()
...
return libcInfo.Version()
```
?

Reply:
I always forget about that pattern when it's my own code :P